CONTRIBUTING.rst

```diff
@@ -0,0 +1,53 @@
+Contributing
+############
+
+Contributions are welcome and much appreciated!
+
+Development
+-----------
+
+To setup `ConferenceScheduler` for local development:
+
+1. Fork the repository on Github `<https://github.com/https://github.com/PyconUK/ConferenceScheduler/fork>`_
+2. Clone your fork to your local machine::
+
+    git clone git@github.com:your_name_here/ConferenceScheduler.git
+
+3. Create a branch for your work::
+
+    git checkout -b name-of-your-branch
+
+4. Commit your changes and push your branch to Github
+
+5. Create a Pull Request at `<https://github.com/PyconUK/ConferenceScheduler/pulls>`_
+
+
+Testing
+-------
+
+ConferenceScheduler includes a comprehensive test suite. Please ensure that you run all the tests before submitting a Pull Request.
+
+To setup your environment for testing:
+
+1. Install the necessary Python libraries::
+
+    pip install -r requirements.txt
```

Comment:
It seems that the `tests_require` setting only specifies what should be installed - it doesn't instruct setup.py to actually go and get them! 

http://stackoverflow.com/questions/4734292/specifying-where-to-install-tests-require-dependencies-of-a-distribute-setupto#7747140